src/data/db_schema.py

```diff
@@ -146,6 +41,9 @@ class Transaction(Base):
     transactionHash: Mapped[str] = mapped_column(String, index=True, unique=True)
 
     #### Fields from base transactions ####
+    ##############
+    # NOOOOOOOOO, transaction shouldn't have a foreign key to poolInfo
+    ##############
```

Comment:
Why not?

Reply:
So hopefully we can make a basic Transaction table that has no dependency on hyperdrive.  We could attach hyperdrive specific transaction info from another table and join when necessary.